sdk/keyvault/Azure.Security.KeyVault.Keys/src/RsaKeyCreateOptions.cs

```diff
@@ -12,22 +12,23 @@ namespace Azure.Security.KeyVault.Keys
 {
     public class RsaKeyCreateOptions : KeyCreateOptions
     {
-        public int KeySize { get; set; }
+        public string Name { get; set; }
         public KeyType KeyType { get; set; }
+        public int KeySize { get; set; }
+        public bool Hsm { get; set; }
 
-        public RsaKeyCreateOptions() 
+        public RsaKeyCreateOptions(bool hsm) 
```

Comment:
Design of this type is a little strange, ctor takes bool turns it into KeyType but KeyType property is writeable.

Reply:
Good point. I will change that